bindings/python/pymongocrypt/binding.py

```diff
@@ -19,7 +19,7 @@
 import cffi
 
 from pymongocrypt.compat import PY3
-from pymongocrypt.version import _MIN_LIBMONGOCRYPT_VERSION
+from pymongocrypt.version import MIN_LIBMONGOCRYPT_VERSION
```

Comment:
I made this public since we use it in the instructions above. I don't think it should be a problem, but do let me know what you think.